python/kfserving/kfserving/server.py

```diff
@@ -53,6 +54,7 @@ def __init__(self, protocol: str = args.protocol, http_port: int = args.http_por
         self.grpc_port = grpc_port
         self.protocol = protocol
         self._http_server: Optional[tornado.httpserver.HTTPServer] = None
+        self.explainer = None
```

Comment:
This feels a little awkward. Can you share your thoughts on encoding these pieces into the KFServer vs wiring it all through the model object?

Reply:
I'll change it so its back in `KFModel`. I think further refactoring could be done in a further PR.